.php-cs-fixer.dist.php

```diff
@@ -119,5 +119,9 @@
         'types_spaces' => true,
         'visibility_required' => true,
         'void_return' => true,
-        'yoda_style' => false,
+        'yoda_style' => [
+            'equal' => true,
+            'identical' => false,
+            'less_and_greater' => null,
+        ],
```

Comment:
See Example2: https://mlocati.github.io/php-cs-fixer-configurator/#version:3.8|fixer:yoda_style
<img width="1130" alt="Screenshot 2022-03-26 at 13 03 40" src="https://user-images.githubusercontent.com/5256287/160238493-9efddcee-9d59-43e1-bb1c-fdef3ea73945.png">

It's only enabled for equality. I didn't know we could have this granular option, which is my favourite for now. What do you say, @jenshaase?